docs/config.adoc

```diff
@@ -458,6 +463,60 @@ The default length limit is 128 keys.
 )
 ----
 
+[[macos-only-macos-dev]]
+=== Macos only: macos-dev
```

Comment:
this section is no longer true after https://github.com/jtroo/kanata/pull/652/commits/07d133b08fc89cb09a3018c45a1969394f834222